scipy/interpolate/__init__.py

```diff
@@ -5,87 +5,115 @@
 
 .. currentmodule:: scipy.interpolate
 
-Sub-package for objects used in interpolation.
+Sub-package for functions and objects used in interpolation.
+
+See the :ref:`user guide <tutorial-interpolate>` for recommendations on choosing a
+routine, and other usage details.
 
-As listed below, this sub-package contains spline functions and classes,
-1-D and multidimensional (univariate and multivariate)
-interpolation classes, Lagrange and Taylor polynomial interpolators, and
-wrappers for `FITPACK <http://www.netlib.org/dierckx/>`__
-and DFITPACK functions.
 
 Univariate interpolation
 ========================
 
 .. autosummary::
    :toctree: generated/
 
-   interp1d
+   make_interp_spline
+   CubicSpline
+   PchipInterpolator
+   Akima1DInterpolator
+   FloaterHormannInterpolator
    BarycentricInterpolator
    KroghInterpolator
-   barycentric_interpolate
-   krogh_interpolate
-   pchip_interpolate
    CubicHermiteSpline
-   PchipInterpolator
-   Akima1DInterpolator
-   CubicSpline
+
+**Low-level data structures for univariate interpolation:**
+
+.. autosummary::
+   :toctree: generated/
+
    PPoly
    BPoly
-   FloaterHormannInterpolator
+   BSpline
 
 
 Multivariate interpolation
 ==========================
 
-Unstructured data:
+**Unstructured data**
 
 .. autosummary::
    :toctree: generated/
 
-   griddata
    LinearNDInterpolator
    NearestNDInterpolator
    CloughTocher2DInterpolator
+   griddata - a convenience wrapper for the above three interpolators
    RBFInterpolator
-   Rbf
    interp2d
 
-For data on a grid:
+**For data on a grid:**
 
 .. autosummary::
    :toctree: generated/
 
-   interpn
    RegularGridInterpolator
-   RectBivariateSpline
+   interpn - a convenience wrapper for `RegularGridInterpolator`
```

Comment:
We seem to be going away from convenience wrappers. How about moving this to additional tools? 

Also, I'm not sure if the stuff after the name of the function renders. I think that will be replaced with the one line summary, so if that's the case (let's check), let's leave it out.

Reply:
Huh. It definitely used to render, but does not seem to anymore. Which part of the doc infra is responsible, I've no idea.